corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -662,6 +664,15 @@ hqDefine('geospatial/js/models', [
             updateSelectedSavedPolygonParam();
         };
 
+        self.exportSelectedPolygonGeoJson = function (data, event) {
+            if (self.activeSavedPolygon()) {
+                const convertedData = 'text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(self.activeSavedPolygon().geoJson));
+                $(event.target).attr('href', 'data:' + convertedData);
+                $(event.target).attr('download','data.geojson');
+                return true;
```

Comment:
The return only happens inside the conditional block. Should we add a default `return false` outside the block?

Reply:
This can be added but not necessary. 
Just for info, the `return true` was added to allow the default click action by knockout which is in this case is navigate to link's href.

Reply:
I think explicitly passing false to block the next event does sound like the right thing to do here.

Additionally, in terms of commits, it would be nice to have the usage of this function in the same commit itself so its visible how the change impacts where this is used. For commits, its good to think of keeping changes that are functionally related, in the same commit.

Reply:
Addressed in 578e18a52acbbc2449ca41c853bcf54a64934383